### PR TITLE
Fixes for build

### PR DESCRIPTION
**Pull request by sebworks**
Fixes for build

## Changes

- Modified `CHANGELOG.md` to fix duplicate entries.
- Modified `cfgov/unprocessed/css/organisms/chart.less` to fix less error.

## Testing

- Run `gulp styles`

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/unprocessed/css/organisms/chart.less [hunk 1]
```diff
@@ -21,7 +21,7 @@
 */
 
 // Import external cfpb-chart-builder stylesheet
-@import (less) 'cfpb-chart-builder.css';
+// @import (less) 'cfpb-chart-builder.css';
```
Comment (contolini) [inline cfgov/unprocessed/css/organisms/chart.less]: Oop -- sorry about this! Is there a correct way to pull in these styles? We'll need them for the charts.

Comment (sebworks) [inline cfgov/unprocessed/css/organisms/chart.less]: naw, I think the issue isn't this reference. The issue is that  `shrinkwrap.json` file never got updated when package.json changed.  If I run  `npm install` the `cfpb-chart-builder` doesn't get installed ( it does on Travis however ).  @contolini, can you verify this? If so, I'll regenerate the `shrinkwrap.json` file in this PR.

Comment (contolini) [inline cfgov/unprocessed/css/organisms/chart.less]: Ah good find. It's updated in this PR but we probably won't merge it until tomorrow https://github.com/cfpb/cfgov-refresh/pull/2712

Possibly related -- that referenced CSS file is in the library's dist directory and I just realized gulp may not be configured to search for it there. https://github.com/cfpb/cfpb-chart-builder/tree/master/dist I'm on mobile and can't investigate ATM but that may also be causing problems.

**Comment by contolini** [comment]
@sebworks FWIW cfgov-refresh builds fine for me. It successfully finds `cfpb-chart-builder.css` in the node_modules directory. Is it not working for you?

**Comment by sebworks** [comment]
Delete your `node_modules` directory and run `frontend_sh`. Do you you get an error when running `gulp styles`? If I run `npm outdated` I get:
screen shot 2017-01-30 at 5 06 49 pm

It will run fine once I run `npm install cfpb-chart-builder` (doesn't have trouble finding `cfpb-chart-builder.css`).  I'm going to remove the modification to file `cfpb-chart-builder.css`. Once the `npm-shrinkwrap.json` file is updated it should run fine.

**Pull request merged**